Don't define a "module" extension for python2 - this breaks PLIST and loading a
bare .so is supported anyway by python2. python3 has SOABI, or rather
EXT_SUFFIX nowadays.

--- mk/python.mk~	2015-06-26 12:32:49.000000000 +0200
+++ mk/python.mk	2023-01-09 17:07:42.302801621 +0100
@@ -6,9 +6,9 @@
 PYPREFIX  := $(shell $(PYTHON) -c 'import sys; sys.stdout.write(sys.exec_prefix.replace("\\","/"))')
 PYINCDIR  := $(shell $(PYTHON) -c 'import sys, distutils.sysconfig; sys.stdout.write(distutils.sysconfig.get_python_inc().replace("\\","/"))')
 
-PythonSHAREDLIB_SUFFIX = $(shell $(PYTHON) -c 'import sys, distutils.sysconfig; sys.stdout.write((distutils.sysconfig.get_config_var("SO") or ".so").lstrip("."))')
+PythonSHAREDLIB_SUFFIX = $(shell $(PYTHON) -c 'import sys, distutils.sysconfig; sys.stdout.write((distutils.sysconfig.get_config_var("EXT_SUFFIX") or ".so").lstrip("."))')
 
-PY_MODULE_SUFFIX := $(shell $(PYTHON) -c 'import sys; sys.stdout.write((sys.hexversion < 0x3000000) and "module" or "")')
+PY_MODULE_SUFFIX :=
 
 PYINCFILE := "<Python.h>"
 PYINCTHRD := "<pythread.h>"
